### ISSUE TITLE
mklfpk.sh doesn't detect downloading errors

### ISSUE THREAD

**Issue by PovelikinRostislav** [intel/daal#169]
**Describe the bug**
No rule to make target '/lnx/lib/intel64/libdaal_vmlipp_core.a' error is shown during DAAL build

**To Reproduce**
1. Setup
1.1 `source /opt/intel/compilers_and_libraries/linux/bin/compilervars.sh intel64`
1.2 `source /opt/intel/tbb/bin/tbbvars.sh intel64`
2. Call `make -f makefile daal PLAT=lnx32e`
3. See error
```
========= Building core =========
make: *** No rule to make target '/lnx/lib/intel64/libdaal_vmlipp_core.a', needed by '__work/lnx32e/daal/lib/libdaal_core.a'.  Stop.
```

**Expected behavior**
Build succeeded

**Environment:**
 - OS: e.g. Ubuntu 18.04
 - Compiler: icpc (ICC) 19.0.5.281 20190815
 - DAAL Version: 2f4ad00fdfadba45769faf46927f6bd165112224

**Comment by dr-pain** [comment]
Hello!
how you did check out?
looks like you missed  step #5 from "Installation from Sources" instruction.
Andrey

**Comment by PovelikinRostislav** [comment]
Hello @dr-pain,
I've missed the call to the `scripts/mklfpk.sh 32e` on description, thank you for the notice.
By the way, in my case `scripts/mklfpk.sh 32e` failed to download via `curl` due to proxy settings, but the script didn't handle it and proceed to the un`tar`ing, failed and finally said that `Downloaded and unpacked Intel(R) MKL small libraries to /home/rpovelik/dev/daal/externals/mklfpk` :)

To sum up, I didn't notice that curl failed to download libraries.
It would be great if we can detect the error of intermediate steps and fail the build.

**Comment by dr-pain** [comment]
we will try to fix mklfpk.sh soon.
we will also try to make diagnostic for this case (now it is "No rule to make target '/lnx/lib/intel64/libdaal_vmlipp_core.a' ") more informative.

Andrey

**Comment by PovelikinRostislav** [comment]
@dr-pain, thanks!

**Comment by vmeshche** [comment]
@PovelikinRostislav issue with mklfpk.sh committed to master. 
https://github.com/intel/daal/pull/170

**Comment by SmirnovEgorRu** [comment]
@PovelikinRostislav, could you, please, confirm that the issue is resolved?

**Comment by PovelikinRostislav** [comment]
Confirmed, fixed

**Comment by dr-pain** [comment]
> we will also try to make diagnostic for this case (now it is "No rule to make target '/lnx/lib/intel64/libdaal_vmlipp_core.a' ") more informative.
https://github.com/intel/daal/pull/247
